mystic-ai/pipeline
# v0.3.0 - Remote files (F15)

# Pull request outline

Use remote files in a new pipeline definition. 

_References feature [F14 Remote PipelineFile Reference](https://outline.mystic.ai/doc/f15-remote-pipelinefile-reference-uG24fLv7Tb)_

## Checklist:

~~- [ ] Check for breaking changes in Resource~~

Added:
- Download remotes files to local cache
- Use remote files in new pipeline definition

Changed:
_none_

Removed:
_none_

## Related issues:
_none_

___

- _You can add a reference to an issue using hash followed by the issue number_
- _If a checklist item is to be ignored it must be struck through_
